lib/rex/powershell/payload.rb

```diff
@@ -13,7 +13,7 @@ def self.read_replace_script_template(template_path, filename, hash_sub)
   end
 
   def self.to_win32pe_psh_net(template_path, code)
-    rig = Rex::RandomIdentifier::Generator.new()
+    rig = Rex::RandomIdentifierGenerator.new()
```

Comment:
the diff reverted this change which is necessary, can you please put this back to Rex::RandomIdentifier::Generator , thanks